src/models/treemodel.cpp

```diff
@@ -54,6 +54,10 @@ QVariant BottomUpModel::headerColumnData(int column, int role) const
                 "The name of the executable the symbol resides in. May be empty when debug information is missing.");
         }
 
+        if (m_results.costs.unit(column - NUM_BASE_COLUMNS) == Data::Costs::Unit::TracepointCost) {
+            return tr("The total time spend in the function.");
```

Comment:
in which function? this should be rephrased to include the typeName (which is the tracepoint)

Reply:
`tr("The total time spend in %1.").arg(m_results.costs.typeName(column - NUM_BASE_COLUMNS))`